app/src/main/res/layout/frag_note.xml

```diff
@@ -1,74 +1,94 @@
                 android:layout_height="wrap_content"
+                android:layout_marginBottom="16dp"
+                android:background="@android:color/transparent"
+                android:hint="@string/prompt_title"
                 android:imeOptions="actionNext"
-                android:inputType="textCapWords"
-                android:maxLines="1" />
-
-        </android.support.design.widget.TextInputLayout>
+                android:inputType="textCapWords|textNoSuggestions"
+                android:maxLines="1"
+                android:textSize="22sp"
+                android:textStyle="bold" />
 
-        <LinearLayout
-            android:id="@+id/tagsRow"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:orientation="horizontal"
-            android:visibility="gone">
-
-            <TextView
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:layout_gravity="center_vertical"
-                android:paddingLeft="4dp"
-                android:text="Tags:" />
-
-            <HorizontalScrollView
-                android:layout_width="0dp"
+            <LinearLayout
+                android:id="@+id/tagsRow"
+                android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:layout_weight="1">
+                android:orientation="horizontal"
+                android:visibility="gone">
 
-                <LinearLayout
-                    android:id="@+id/tagsLayout"
+                <TextView
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
-                    android:orientation="horizontal"
-                    />
+                    android:layout_gravity="center_vertical"
+                    android:paddingLeft="4dp"
+                    android:text="Tags:" />
+
+                <HorizontalScrollView
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_weight="1">
+
+                    <LinearLayout
+                        android:id="@+id/tagsLayout"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:orientation="horizontal" />
+
+                </HorizontalScrollView>
 
-            </HorizontalScrollView>
+            </LinearLayout>
+
+            <EditText
+                android:id="@+id/bodyEdit"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:background="@android:color/transparent"
+                android:gravity="top"
+                android:hint="@string/prompt_text"
+                android:inputType="textMultiLine|textNoSuggestions"
```

Comment:
Ah, yeah fair enough.  Mostly I was just getting annoyed at how ugly the spellcheck underline is, but that shouldn't be a problem when I'm not just typing nonsense to test.